cmd/earthly/main.go

```diff
@@ -2139,8 +2159,36 @@ func (app *earthlyApp) actionPrune(c *cli.Context) error {
 	return nil
 }
 
+func (app *earthlyApp) actionDocker(c *cli.Context) error {
+	app.commandName = "docker"
+
+	dir := filepath.Dir(app.dockerfilePath)
+	earthfilePath := filepath.Join(dir, "Earthfile")
+	if fileutil.FileExists(earthfilePath) {
+		return errors.Errorf("earthfile already exists; please delete it if you wish to continue")
+	}
+	defer os.Remove(earthfilePath)
+
+	err := docker2earthly.Docker2Earthly(app.dockerfilePath, earthfilePath, app.earthfileFinalImage)
+	if err != nil {
+		return err
+	}
+
+	app.interactiveDebugging = true
+	flagArgs := []string{}
+	nonFlagArgs := []string{"+build"}
+
+	return app.actionBuildImp(c, flagArgs, nonFlagArgs)
+}
+
 func (app *earthlyApp) actionDocker2Earthly(c *cli.Context) error {
-	return docker2earthly.Docker2Earthly(app.dockerfilePath, app.earthfilePath, app.earthfileFinalImage)
+	app.commandName = "docker2earthly"
+	err := docker2earthly.Docker2Earthly(app.dockerfilePath, app.earthfilePath, app.earthfileFinalImage)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(os.Stderr, "An Earthfile has been generated; to run it use: earthly +build; then run with docker run -ti %s\n", app.earthfileFinalImage)
```

Comment:
Should we standardize on using `app.console` for messages to the user? Perhaps console should just always print to stderr.

Reply:
Makes sense - can't remember if it does already. In any case, it should be easy to switch in the `conslogging` package.